src/Components/cardHolder.css

```diff
@@ -0,0 +1,15 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather&display=swap');
```

Comment:
should put this in `theme`, can be removed alongside google font imports in `index.html`